### ISSUE TITLE
Segmentation Fault when using pubkeyhash

### ISSUE THREAD

**Issue by IsurangaPerera** [citp/BlockSci#169]
I tried to get a cluster using pubkeyhash address and got a segmentation fault. Is this an issue

**Comment by maltemoeser** [comment]
It's hard to tell what's going wrong with so little information. Could you provide the code that's causing the segfault?

**Comment by maltemoeser** [comment]
The address you're looking at is in a supercluster with a few hundred million addresses. Does the same happen with other addresses?

**Comment by IsurangaPerera** [comment]
Yes, tried with several pubkeyhash addresses. But worked fine with scripthash addresses

**Comment by maltemoeser** [comment]
These are all in the same cluster. Do P2PKH addresses fail for just this cluster and P2SH addresses work for this cluster, or do P2PKH addresses fail in general?

E.g., does `1ACTjRT8dSCfm4s5ApXA2bn2rZJkUwedqE` (P2PKH in other cluster) work? Does `3FJ3nKWp44zmoVXjFBwrFUyMYALXuTmauf` (P2SH in same cluster if you used the default clustering) work?

**Comment by IsurangaPerera** [comment]
`1ACTjRT8dSCfm4s5ApXA2bn2rZJkUwedqE` works fine. But the same problem for P2SH (`3FJ3nKWp44zmoVXjFBwrFUyMYALXuTmauf`)
